Makefile.core.mk

```diff
@@ -307,6 +307,7 @@ update-istio: ## Updates the Istio commit hash in the 'latest' entry in versions
 ## Location to install dependencies to
 LOCALBIN ?= $(shell pwd)/bin
 $(LOCALBIN):
+	echo "mkdir -p $(LOCALBIN)"
```

Comment:
This is redundant, I think. The command below is echoed anyway because there's no `@` at the beginning to mute it.

Reply:
yep np missed that one